src/libs/initialFetch.js

```diff
@@ -0,0 +1,101 @@
+import API from "@aws-amplify/api";
+import sortby from "lodash.sortby";
+import sanity from "./sanity";
+
+export async function fetchStarterKitExperience(id) {
```

Comment:
I refactored all of these queries to return objects instead of directly attempting to update state -- that way, after all the promises have resolved and the data is in, state only needs to be updated a single time. 

In functional components if an object is used with useState you need to pass the _entire_ object when updating (`const [state, setState] = useState({...})` is the equivalent of `this.state = {state: {...}}` in a class component, if that makes sense), so the way that React batches state updates can cause unintended effects and unanticipated overwriting.

I should be able to abstract this whole file further in a future PR, there's a fair amount of repeated code across all of the functions.

Reply:
This is a great idea, thank you! Will make it easier to update/refresh the knowledge base in the future, especially because I would like to move it to a cached system for universal search.